test/swagger/constraints_no_validations.json

```diff
@@ -0,0 +1,210 @@
+{
```

Comment:
why do we need 2 different json files? what's the difference between them? Could we generate for one with the client_side_validation flag set to false and again for the same with the flag set to true?

Reply:
Initially, I wanted to do it in one file. But, that is causing some errors in loading the required modules because of the names of the files.. (Remember the loading issue we found during the profile work). So, instead of complicating the logic just to reduce the one extra file, I created another file. Also, since they are separate test case, the test design principle suggests that they must be independent in all aspects.  :-)

Reply:
ok, that's fine then.